CHANGELOG.md

```diff
@@ -16,12 +16,13 @@ Documentation for rocRAND is available at
 ### Changed
 
 * `rocrand_discrete` for MTGP32, LFSR113 and ThreeFry generators now uses the alias method, which is faster than binary search in CDF.
+* Removed TBB dependency for multi-core processing of host-side generation.
```

Comment:
This PR will be going in for ROCm 6.4 - please add a new release entry in the changelog, and also bump the version number in CMakeLists.txt to match.  Thanks!